ui-v2/tests/unit/mixins/acl/with-actions-test.js

```diff
@@ -18,3 +28,64 @@ test('it works', function(assert) {
   const subject = this.subject();
   assert.ok(subject);
 });
+test('use persists the token and calls transitionTo correctly', function(assert) {
+  assert.expect(4);
+  this.register(
+    'service:feedback',
+    Service.extend({
+      execute: function(cb, name) {
+        assert.equal(name, 'use');
+        return cb();
+      },
+    })
+  );
+  const item = { ID: 'id' };
+  this.register(
+    'service:settings',
+    Service.extend({
+      persist: function(actual) {
+        assert.equal(actual.token, item.ID);
+        return Promise.resolve(actual);
+      },
+    })
+  );
+  const subject = this.subject();
+  const expected = 'dc.services';
+  const transitionTo = this.stub(subject, 'transitionTo').returnsArg(0);
+  return subject.actions.use
+    .bind(subject)(item)
```

Comment:
You should be able to do `return subject.send('use', item).then` here.

Reply:
I’m trying to just test that one function, to my understanding using `send` would broaden the scope (integration vs unit), does that make sense?